protocol/tbdex_schema_examples.js

```diff
@@ -0,0 +1,50 @@
+let offering = {
+  "pair": "BTC_USD",
+  "unitPrice": 27000.00,
+  "fee": 1.00,
+  "min": 10.00,
+  "max": 100.00,
+  "presentationRequest": { },
+  "payinInstruments": [{
+      "kind": "DEBIT_CARD",
+      "fee": 1,
+      "presentationRequest": {},
+  }],
+  "payoutInstruments": [{
+      "kind": "BTC_ADDRESS",
+      "presentationRequest": {}
+  }]
+}
+
+let rfq = {
+  "rfqId": "1",
+  "pair": "BTC_USD",
+  "amount": 10.00,
+  "verifiablePresentation": { },
+  "payin_instrument": {
+      "kind": "DEBIT_CARD"
+  },
+  "payout_instrument": {
+      "kind": "BTC_ADDRESS"
+  }
+}
+
+let quote = {
+  "rfqId": "1",
+  "quoteId": "2",
+  "expiryTime": "2023-04-14T12:12:12Z",
+  "totalFee": 1.00,
```

Comment:
we're just assuming that Alice will be ok with fee being taken off from the original $10. what if she wanted to pay the fee on top of $10? what if the PFI wanted to offer that?

Reply:
I think most things I have used - it just comes out of it. Never had a great experience though.